tests/unit/test_middleware/test_base_authentication_middleware.py

```diff
@@ -225,3 +226,36 @@ def east_handler() -> None:
 
         response = client.get("/west")
         assert response.status_code == HTTP_403_FORBIDDEN
+
+
+@pytest.mark.noww
```

Comment:
I suppose that wasnt meant to be in the PR or if it is I dont get why, otherwise lgtm

Reply:
Oops I forgot to remove them. Thanks for pointing it out :)